6_Extending_the_Toolkit-Part2.ipynb

```diff
@@ -8,203 +8,7 @@
     }
```

Comment:
<p>This is great!</p><p><br></p><p>One more screenshot / discussion point would be helpful: the CDash page for the pull request... then C++ warnings, build errors, tests can be explained when you discuss it.</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Makes sense</p>